src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateSplit = date.split(fromFormat[3]);
+  const newDate = [];
+
+
+  for (let i = 0; toFormat.length - 1 > i; i++) {
+    const format = toFormat[i];
+    console.log(format)
+    console.log(newDate)
+
+    if (format === 'YYYY' && fromFormat.includes('YY')) {
+      let year = dateSplit[fromFormat.indexOf('YY')]
+      if (year < 30) {
```

Comment:
The comparison `year < 30` may not work as expected because `year` is a string and the comparison is made against a number. Explicitly convert `year` to a number before comparison.